homeassistant/components/avea/light.py

```diff
@@ -0,0 +1,97 @@
+"""Support for the Elgato Avea lights."""
+import logging
+
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS, ATTR_HS_COLOR, SUPPORT_BRIGHTNESS,
+    SUPPORT_COLOR, Light)
+from homeassistant.exceptions import PlatformNotReady
+import homeassistant.util.color as color_util
+
+
+_LOGGER = logging.getLogger(__name__)
+
+SUPPORT_AVEA = (SUPPORT_BRIGHTNESS | SUPPORT_COLOR)
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Set up the Avea platform."""
+    import avea
+
+    nearby_bulbs = avea.discover_avea_bulbs()
+    try:
+        for bulb in nearby_bulbs:
+            bulb.get_name()
+            bulb.get_brightness()
+    except Exception:
```

Comment:
Could you make this except type more specific? Usually catching a generic `Exception` is discouraged. If this uses python's standard APIs, you can probably also use `OSError`.

Reply:
So this component is using the [python avea lib](https://github.com/k0rventen/avea/) which has no own exception implemented. But it depends on [bluepy](https://github.com/IanHarvey/bluepy/) which has an own class [BTLEException](https://github.com/IanHarvey/bluepy/blob/53ce2f2388a936663b94f00636cc2e3677325182/bluepy/btle.py#L37)
Should I use this Exception instead of OSError?